Project.toml

```
@@ -1,7 +1,31 @@
 name = "SPECTrecon"
 uuid = "ab1be465-a7f0-4423-9048-0ee774b70ed9"
-authors = ["Jeff Fessler and group"]
-version = "0.1.0"
+authors = ["Zongyu Li and Jeff Fessler and group"]
+version = "0.0.1"
+
+[deps]
+AbstractFFTs = "621f4979-c628-5d54-868e-fcf4e3e8185c"
+BenchmarkTools = "6e4b80f9-dd63-53aa-95a3-0cdb28fa8baf"
+FFTW = "7a1cc6ca-52ef-59f5-83cd-3a7055c09341"
+ImageFiltering = "6a3955dd-da59-5b1f-98d4-e7296123deb5"
+ImageTransformations = "02fcd773-0e25-5acc-982a-7f6622650795"
+InterpolationKernels = "16730964-a2ec-11e9-36fa-47a4f82bfac6"
+Interpolations = "a98d9a8b-a2ab-59e6-89dd-64a1c18fca59"
+LazyAlgebra = "aeaa251e-9d76-11e9-311b-83636bf44738"
+LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+LinearInterpolators = "2fc109c4-9d77-11e9-32ff-e5c650334c16"
+MAT = "23992714-dd62-5051-b70f-ba57cb901cac"
+MIRTjim = "170b2178-6dee-4cb0-8729-b3e8b57834cc"
+OffsetArrays = "6fe1bfb0-de20-5000-8ca7-80f57d26f881"
+Plots = "91a5bcdd-55d7-5caf-9e0b-520d859cae80"
```

```suggestion
FFTW = "7a1cc6ca-52ef-59f5-83cd-3a7055c09341"
ImageFiltering = "6a3955dd-da59-5b1f-98d4-e7296123deb5"
ImageTransformations = "02fcd773-0e25-5acc-982a-7f6622650795"
InterpolationKernels = "16730964-a2ec-11e9-36fa-47a4f82bfac6"
Interpolations = "a98d9a8b-a2ab-59e6-89dd-64a1c18fca59"
LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
LinearInterpolators = "2fc109c4-9d77-11e9-32ff-e5c650334c16"
OffsetArrays = "6fe1bfb0-de20-5000-8ca7-80f57d26f881"
```
the only packages to put here are the ones for src/